### DISCUSSION TITLE
How are Images stored?

### DISCUSSION THREAD

**Discussion by Peanutguyy** [Chevereto/Chevereto-Free#165] [Q&A]
Hey,

Suppose I started with uploading a zillion of images with the direct option to /images.
And later because the folder got so heavy to just open it I decided to change the images being uploaded to folders by dates.
If one day I decide that I want to merge everything again into one folder (/images), taking all the images from the date folders and moving them to the /images, will they work? or will the server not make them show up because database thinks they are in the folders according to the date?

EDIT: Ok, I see now that when moving images from a date folder to the main image folder that it wont show the image anymore.
I guess it's too impossible/too time consuming to change that?

One more question: When the images in album that the privacy is set to ANYONE with this link only privacy setting that can see the album. will the album/images be indexed on google in any form?

**Comment by rodber** [comment]
Images are stored in the filesystem and the relations are in MySQL database, which only maps the images to their storage location.

You can do whatever you want with the images in the filesystem long as you also update the database records.
